Sources/SegmentFirebase/FirebaseDestination.swift

```diff
@@ -103,9 +108,9 @@ public class FirebaseDestination: DestinationPlugin {
     public func screen(event: ScreenEvent) -> ScreenEvent? {
         
         if let eventName = event.name {
-            FirebaseAnalytics.Analytics.logEvent(FirebaseAnalytics.AnalyticsEventScreenView,
-                                                 parameters: [FirebaseAnalytics.AnalyticsParameterScreenName: eventName])
-            analytics?.log(message: "Firebase setScreenName \(eventName)")
```

Comment:
We probably shouldn't change their spacing

Reply:
Sorry, I was using our spacing, that's why. It's fixed already